etc/profile-m-z/termshark.profile

```diff
@@ -0,0 +1,12 @@
+# Firejail profile for termshark
+# Description: Terminal UI for tshark, inspired by Wireshark
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include termshark.local
+# Persistent global definitions
+# added by included profile
+#include globals.local
+
+# Redirect
```

Comment:
We could block x11/Wayland stuff right?

Reply:
Yeah. In fact we can block ${RUNUSER} completely. Thanks for the pointer!